client/lib/plans/add-ons-list.tsx

```diff
@@ -0,0 +1,44 @@
+import {
```

Comment:
**Note:**

It didn't feel quite right adding data about add ons into the plans features list ( features are automatically included with the plan ), so I created a separate plans add ons list. Totally open to suggestions though

Reply:
We should always prefer existing packages than local Calypso folders. Should this live under `@automattic/calypso-products` instead maybe?

Reply:
Sounds good 👍  I put it here to live as a sibling of the `features-list.jsx` https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/plans/features-list.tsx, but I'm happy to scoot it over.

Calypso products seems like a good spot to me as well

Reply:
Ideally, I'd hope for everything plans-related to migrate and be centralized under the plans data store (that is also on NPM). I wrote some thoughts on this in pdvytD-bG-p2 (stuff under "notes on principles"). But for the moment, `@automattic/calypso-products` sounds like the place to dump them all in 🤷‍♂️

Reply:
> Ideally, I'd hope for everything plans-related to migrate and be centralized under the plans data store (that is also on NPM). I wrote some thoughts on this in pdvytD-bG-p2 (stuff under "notes on principles").

Thanks for sharing --  interesting read, and good info on package-first development in Calypso. I'm currently wrapping my head around how keeping the plans grid as "dumb" as possible will modify how I approach the subsequent PR I'm drafting https://github.com/Automattic/wp-calypso/pull/79104 🤔 Will also give this more thought

Reply:
Moved everything over to @automattic/calypso-products in [e2d1f13](https://github.com/Automattic/wp-calypso/pull/79041/commits/e2d1f13f400d2a4d8bd93b7493a13c29131ba164)